crates/bevy_image/src/image_loader.rs

```diff
@@ -12,6 +12,135 @@ pub struct ImageLoader {
 }
 
 impl ImageLoader {
+    /// Number of image formats, used for computing other constants.
+    const COUNT: usize = {
+        let mut count = 0;
+        #[cfg(feature = "avif")]
+        {
+            count += 1;
+        }
+        #[cfg(feature = "basis-universal")]
+        {
+            count += 1;
+        }
+        #[cfg(feature = "bmp")]
+        {
+            count += 1;
+        }
+        #[cfg(feature = "dds")]
+        {
+            count += 1;
+        }
+        #[cfg(feature = "ff")]
+        {
+            count += 1;
+        }
+        #[cfg(feature = "gif")]
+        {
+            count += 1;
+        }
+        #[cfg(feature = "ico")]
+        {
+            count += 1;
+        }
+        #[cfg(feature = "jpeg")]
+        {
+            count += 1;
+        }
+        #[cfg(feature = "ktx2")]
+        {
+            count += 1;
+        }
+        #[cfg(feature = "pnm")]
+        {
+            count += 1;
+        }
+        #[cfg(feature = "png")]
+        {
+            count += 1;
+        }
+        #[cfg(feature = "qoi")]
+        {
+            count += 1;
+        }
+        #[cfg(feature = "tga")]
+        {
+            count += 1;
+        }
+        #[cfg(feature = "tiff")]
+        {
+            count += 1;
+        }
+        #[cfg(feature = "webp")]
+        {
+            count += 1;
+        }
+        count
+    };
+
+    /// Full list of supported formats.
+    pub const SUPPORTED: &'static [ImageFormat] = &[
```

Comment:
Nit: just "supported" isn't as clear in relation to `ImageLoader` versus `ImageFormat`.

```suggestion
    pub const SUPPORTED_FORMATS: &'static [ImageFormat] = &[
```

Reply:
Great point! Done.